### PR TITLE
Node Beginner Booster Deck

### PR DESCRIPTION
**Pull request by lngchn**

### CODE REVIEW

File: decks/node-beginner/meeting-1.mdx [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+<CodeLayout>
+
+## Setup 🛠
+
+1. Do the Boosters [Initial Setup](https://github.com/Accenture/digital-products-boosters#initial-setup) if you have not done so.
+
+2. From the root of the `digital-products-boosters` repository, run:
+
+```bash
+$ cd boosters/react-beginner/meeting-1
```
Comment (chao-henry) [inline decks/node-beginner/meeting-1.mdx]: Directory path should change to `boosters/node-beginner/meeting-1` if there is a project there to use for the exercises. If not, we can remove this setup page, or re-word it accordingly.

File: decks/node-beginner/meeting-1.mdx [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+
+# Discussion Questions 🤔
+
+_To be discussed during the meeting_
+
+---
+
+# Discussion Question #1
+
+```JavaScript
```
Comment (chao-henry) [inline decks/node-beginner/meeting-1.mdx:81]: We'll want to wrap the page in a <CodeLayout> block to get it to fit correctly.

**Review by chao-henry — CHANGES_REQUESTED** [review]

File: decks/node-beginner/meeting-2.mdx [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    </ol>
+  </div>
+</Split>
+
+---
+
+<CodeLayout>
+
+## Setup 🛠
+
```
Comment (chao-henry) [inline decks/node-beginner/meeting-2.mdx:35]: Might be good to wrap this with <CodeLayout> for smaller screen sizes.

File: decks/node-beginner/meeting-2.mdx [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+1. Install Postman
```
Comment (chao-henry) [inline decks/node-beginner/meeting-2.mdx]: Add link to Postman?

File: decks/node-beginner/meeting-2.mdx [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+# Exercise 4
+
+1. Get a list of all repositories of person next to you.
+2. Sort by create date in descending order.
+
+<MinuteTimer minutes={5} />
+
+---
+
+# Exercise 4 - Answer
```
Comment (chao-henry) [inline decks/node-beginner/meeting-2.mdx:126]: The alignment seems off on this page.

File: decks/node-beginner/meeting-2.mdx [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+
+Need help? https://developer.github.com/v3/search/#search-commits
+
+<MinuteTimer minutes={5} />
+
+---
+
+# Exercise 6 - Answer
+
+- GET https://api.github.com/search/commits?q='delete OR ignore'
```
Comment (chao-henry) [inline decks/node-beginner/meeting-2.mdx]: We switch from numbered list to bullet list for the answers slide. Lets keep it consistent across all slides.

**Review by chao-henry — CHANGES_REQUESTED** [review]

File: decks/node-beginner/meeting-2.mdx (hunk 6 above)
Comment (lngchn) [reply decks/node-beginner/meeting-2.mdx]: I think it's numbered list for question and bullet for answer?

Comment (chao-henry) [reply decks/node-beginner/meeting-2.mdx]: There's an odd formatting in the query string and how it shows up in Markdown.

**Review by chao-henry — APPROVED** [review]

**Pull request merged**